packages/components/src/toolbar-button/index.js

```diff
@@ -12,63 +12,55 @@ import { useContext } from '@wordpress/element';
  * Internal dependencies
  */
 import Button from '../button';
+import ToolbarItem from '../toolbar-item';
 import ToolbarContext from '../toolbar-context';
-import AccessibleToolbarButtonContainer from './accessible-toolbar-button-container';
 import ToolbarButtonContainer from './toolbar-button-container';
 
 function ToolbarButton( {
 	containerClassName,
-	icon,
-	title,
-	shortcut,
-	subscript,
-	onClick,
 	className,
-	isActive,
-	isDisabled,
 	extraProps,
 	children,
+	...props
```

Comment:
Removed most of the props here so we can spread `{ ...props }` directly into `Button`. This way, `ToolbarButton` (when used within `<Toolbar __experimentalAccessibilityLabel="label">`) and `Button` have identical API.